aspnetcore/web-api/advanced/formatting/3.0sample/Startup.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace ResponseFormattingSample
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        #region snippet
```

Comment:
Not trying to recommend a thing, but I just vastly prefer using inline code snippets to samples like this. It's really difficult to tell what a code snippet is referring to when it's separated like this.

Reply:
That's why you always need a window open to the [Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/web-api/advanced/formatting?view=aspnetcore-2.2&branch=pr-en-us-12077)